src/components/YourCompany/YourCompany.vue

```diff
@@ -490,43 +484,35 @@ export default class YourCompany extends Mixins(
     const haveChanges = (
       this.companyNameChanges ||
       this.companyTypeChanges ||
-      this.contactInfoChanges ||
       this.nameTranslationChanges ||
-      this.folioNumberChanges ||
       this.officeAddressChanges
     )
     this.setDefineCompanyStepChanged(haveChanges)
     this.emitHaveChanges(haveChanges)
   }
 
-  /** Emits Have Changes event. */
+  /** Emits Have Changes event (only used by corrections). */
   @Emit('haveChanges')
   private emitHaveChanges (val: boolean): void {}
 
-  /** Updates store initially and when local isEditingName property has changed. */
+  /** Updates store initially and when isEditingName property has changed. */
   @Watch('isEditingNames', { immediate: true })
   private onEditingNameChanged (val: boolean): void {
     this.setValidComponent({ key: 'isValidCompanyName', value: !val })
     this.setEditingCompanyName(val)
   }
 
-  /** Updates store initially and when local isEditingType property has changed. */
+  /** Updates store initially and when isEditingType property has changed. */
   @Watch('isEditingType', { immediate: true })
   private onEditingTypeChanged (val: boolean): void {
     this.setValidComponent({ key: 'isValidBusinessType', value: !val })
   }
 
-  /** Updates store initially and when local isEditingTranslations property has changed. */
+  /** Updates store initially and when isEditingTranslations property has changed. */
   @Watch('isEditingTranslations', { immediate: true })
   private onEditingTranslationChanged (val: boolean): void {
     this.setValidComponent({ key: 'isValidNameTranslation', value: !val })
   }
-
-  /** Updates store initially and when local isEditingFolio property has changed. */
-  @Watch('isEditingFolio', { immediate: true })
-  private onEditingFolioChanged (val: boolean): void {
```

Comment:
This method is now in FolioInformation.vue.
(I'm trying to hide such details in sub-components.)